And/modify csv feature

andrey-canon opened the pull request:
## Description
This branch implements a new view to show the result state for every row in the csv file, also allows to use a file with headings, from now the file must be contain the next headings (Student, TeamTopic, TeamName, TeamCountry, TeamLanguage, TeamDescription )

## Reviewers 
- [ ] @nicovanniekerk 
- [x] @jfavellar90 
- [ ] @diegomillan

andrey-canon committed:
Fixing bug with cancel button

andrey-canon committed:
Add a new view to show errors and success

jfavellar90 commented on lms/djangoapps/teams/static/teams/js/teams_features.js:
```diff
@@ -61,19 +61,43 @@
+                    var list = $("<ul></ul>");
+
+                    for (var key in errors){
+                        var item = $("<li style='color: red; font-size:12px;'>Error in "+key+" = "+errors[key]+"</li>");
```
what do you think about using inline css styles @andrey-canon @diegomillan ? Can Adonis give us a hand here?

andrey-canon replied:
check, it was removed

jfavellar90 commented on lms/djangoapps/teams/static/teams/js/teams_features.js:
```diff
@@ -61,19 +61,43 @@
-                done:function(){
+                done:function(e, data){
                     $(".page-header-secondary").empty();
                     buttonAddMembers(staff, url);
```
Sorry don't understand the logic behind calling the method buttonAddMembers inside itself

andrey-canon replied:
in the previous version was necessary to reload the button because it only works once time, but now This line is not necessary and was removed in last commit

jfavellar90 commented on lms/djangoapps/teams/static/teams/js/teams_features.js:
```diff
@@ -61,19 +61,43 @@
                 },
                 fail: function(e, data){
+                    $(".page-header-secondary").empty();
+                    buttonAddMembers(staff, url);
```
Sorry don't understand the logic behind calling the method buttonAddMembers inside itself

andrey-canon replied:
in the previous version was necessary to reload the button because it only works once time, but now This line is not necessary and was removed in last commit

jfavellar90 commented on lms/djangoapps/teams/views.py:
```diff
@@ -1277,13 +1277,12 @@ class CreateTeams(GenericAPIView):
+            return Response(self.handle_uploaded_file(file['fileUpload'], course_id))
+        return Response(status=status.HTTP_400_BAD_REQUEST)
 
     def handle_uploaded_file(self, file, course_id):
```
@andrey-canon @diegomillan do we need to worry about the fact that someone can upload a file with a lot of lines (it can cause an slow response as it's a synchronous process)?

andrey-canon replied:
Don't worry

andrey-canon committed:
Addressing PR comments
-Remove styles
-Remove unnecessary calls
-Adding the field_errors when a team is created

jfavellar90 commented:
if @diegomillan agree, merge this one
